src/components/CreateCoursePage/CreateCourseForm.scss

```diff
@@ -0,0 +1,19 @@
+.create-course-form {
```

Comment:
Shouldn't this file be included somewhere or is it picked automatically by loader?

Reply:
We are including it in `CreateCourseForm.jsx`.